docs/config.json

```diff
@@ -1286,15 +1286,15 @@
       "last_report": "August 9th, 2022"
     },
     "teleport": {
-      "major_version": "11",
-      "version": "11.0.3",
+      "major_version": "12",
+      "version": "12.0",
       "golang": "1.19",
       "plugin": {
         "version": "11.0.3"
```

Comment:
Don't forget plugins updates too.

Reply:
Will they be released at the same time with the same version, `12.0`?

Reply:
Yep. Plugins are aligned to the latest release branch and get released immediately after we cut a teleport release.

Reply:
```suggestion
        "version": "12.0.0"
```